projects/the-application/src/assets/i18n/main/bn.json

```diff
@@ -1,17 +1,17 @@
 {
-  "ADD_TO_HOME_SCREEN": "ADD_TO_HOME_SCREEN",
-  "CLOSE": "CLOSE",
-  "DESCRIPTION": "DESCRIPTION",
-  "FULL_STOP": "FULL_STOP",
-  "GAME": "GAME",
-  "HIGH_SCORES": "HIGH_SCORES",
-  "INSTALL": "INSTALL",
-  "INSTALL_MEMORY_GAME": "INSTALL_MEMORY_GAME",
-  "PLEASE_RELOAD": "PLEASE_RELOAD",
-  "PLEASE_FINISH_OR_RESET_GAME": "PLEASE_FINISH_OR_RESET_GAME",
-  "RECENT_SCORES": "RECENT_SCORES",
-  "REFRESH": "REFRESH",
-  "SETTINGS": "SETTINGS",
-  "TITLE": "TITLE",
-  "UPDATE_AVAILABLE": "UPDATE_AVAILABLE"
+  "ADD_TO_HOME_SCREEN": "হোম পর্দায় যোগ করুন",
+  "CLOSE": "বন্ধ",
+  "DESCRIPTION": "বর্ণনা",
+  "FULL_STOP": "ফুলস্টপ্",
+  "GAME": "খেলা",
+  "HIGH_SCORES": "সেরা স্কোর",
+  "INSTALL": "ইনস্টল",
+  "INSTALL_MEMORY_GAME": "মেমরি গেম ইনস্টল করুন",
+  "PLEASE_RELOAD": "লোড করুন",
+  "PLEASE_FINISH_OR_RESET_GAME": "খেলা শেষ বা পুনরায় সেট করুন",
+  "RECENT_SCORES": "সাম্প্রতিক স্কোর",
+  "REFRESH": "রিফ্রেশ",
+  "SETTINGS": "সেটিংস",
```

Comment:
Can you add TAP and THEN?

See https://github.com/tedaky/memory-game/pull/16/files#diff-47c31de70082a49e837b88ef47ba61dc365e3d92008c6d09cbacc1a0edbfb98a

Reply:
added